src/safariservices.cs

```diff
@@ -456,4 +467,38 @@ interface SFUniversalLink {
 		bool Enabled { [Bind ("isEnabled")] get; set; }
 	}
 #endif
+
+	[Static]
+	[iOS (15,0), Mac (11,0), MacCatalyst (15,0), NoTV, NoWatch]
+	[BaseType (typeof (NSObject))]
```

Comment:
this should work without being an `NSObject` subclass
since those have impact (registration) it's better to avoid them, unless needed

Reply:
Okay! What is a way to know when to use them? Do I not use it in this interface just contains fields?

Reply:
If it's a `static` type and it's **not** a type that exists in Apple SDK then assume, by default, that it's not needed (to have it subclass `NSObject`).